content/en/docs/reference/dockershim-removal/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Dockershim Removal and CRI-compatible Runtimes
```

Comment:
How I intended it

```suggestion
title: Node Reference Information
```
_and_
change the path:
```bash
mv content/en/docs/reference/dockershim-removal content/en/docs/reference/node/
```


If we have future node-related references (I expect that we will), they can also live in here.